### PR TITLE
Rename ssl

### PR DESCRIPTION
**Pull request by dishitpala**
Rename `ssl` command to `ssl-verify`
and also changed function name from `ssl` to `ssl_verify`

### CODE REVIEW

File: src/helper/class-ee-site.php [hunk 1]
```diff
@@ -945,11 +948,9 @@ public function ssl( $args = [], $assoc_args = [], $www_or_non_www = false ) {
 			$is_solver_dns   = ( $this->site_data['site_ssl_wildcard'] || 'dns' === $preferred_challenge ) ? true : false;
 			$api_key_present = ! empty( get_config_value( 'cloudflare-api-key' ) );
 
-			$warning = ( $is_solver_dns && $api_key_present ) ? "The dns entries have not yet propogated. Manually check: \nhost -t TXT _acme-challenge." . $this->site_data['site_url'] . "\nBefore retrying `ee site ssl " . $this->site_data['site_url'] . "`" : 'Failed to verify SSL: ' . $e->getMessage();
+			$warning = ( $is_solver_dns && $api_key_present ) ? "The dns entries have not yet propogated. Manually check: \nhost -t TXT _acme-challenge." . $this->site_data['site_url'] . "\nBefore retrying `ee site ssl-verify " . $this->site_data['site_url'] . "`" : 'Failed to verify SSL: ' . $e->getMessage();
 			EE::warning( $warning );
-			EE::warning( sprintf( 'Check logs and retry `ee site ssl %s` once the issue is resolved.', $this->site_data['site_url'] ) );
-
-			return;
```
Comment (mrrobot47) [inline src/helper/class-ee-site.php]: Need this `return;` statement.

**Comment by kirtangajjar** [comment]
@dishitpala Can you pull the latest code and see if there are any invocations of `ssl` command that needs to be fixed? Also can you see if any function in https://github.com/EasyEngine/site-type-wp/ and https://github.com/EasyEngine/site-type-php/ needs updating?

If these things are done then this PR LGTM

**Comment by kirtangajjar** [comment]
> @dishitpala Can you pull the latest code and see if there are any invocations of `ssl` command that needs to be fixed? Also can you see if any function in https://github.com/EasyEngine/site-type-wp/ and https://github.com/EasyEngine/site-type-php/ needs updating?
> 
> If these things are done then this PR LGTM

I've checked. There's no other place where this function is getting called. This PR LGTM and is ready to merge.

**Review by kirtangajjar — APPROVED** [review]

**Comment by kirtangajjar** [comment]
I've tested the PR on a test server and SSL verification was working fine with both the commands, so we don't have to worry about backward compatibility:
```
ee site ssl example.com
ee site ssl-verify example.com
```

**Pull request merged**